### PR TITLE
start validating code-scanning workflows

### PR DESCRIPTION
**Pull request by nickfyson**
This PR explicitly enables the validation for Code Scanning workflows.

👋 @aparna-ravindra Would you mind taking a look at this? I see you've been active with these scripts, so are probably well placed to review! 🙂

### CODE REVIEW

File: script/validate-data/index.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ async function checkWorkflows(folders: string[]): Promise<WorkflowWithErrors[]>
     });
 
     for (const e of dir) {
-      if (e.isFile()) {
+      if (e.isFile() && extname(e.name) === ".yml") {
```
Comment (aparna-ravindra) [inline script/validate-data/index.ts:52]: This looks good to me. Would you like to check for both `.yml` and `.yaml` here?

Comment (nickfyson) [reply script/validate-data/index.ts]: 👍 will do!

**Review by anaarmas — APPROVED** [review]
approving based on previous approval + diff since

**Comment by itepopup11** [comment]
nukadu

**Comment by itepopup11** [comment]
> This PR explicitly enables the validation for Code Scanning workflows.
> 
> 👋 @aparna-ravindra Would you mind taking a look at this? I see you've been active with these scripts, so are probably well placed to review! 🙂

**Pull request merged**